continuous-deployment/pom.xml

```diff
@@ -40,4 +40,7 @@
         </plugins>
     </build>
 
+    <properties>
+        <project.Automatic-Module-Name>io.cucumber.continuous.deployment</project.Automatic-Module-Name>
```

Comment:
This module is excluded from release. It doesn't need the module name.

Reply:
changed